apps/rewards/app/utils/helpers.js

```diff
@@ -4,7 +4,8 @@ import {
   ETH_DECIMALS_NUMBER,
   RECURRING_DIVIDEND,
 } from './constants'
-import doubleCheck from '../assets/double-check.svg'
+import doubleCheck
+  from '../../../../shared/ui/components/assets/svg/double-check.svg'
```

Comment:
Hmm, I see why we stuck with exporting the raw `svg` rather than a component – it's what we were already doing in Rewards. It looks like this is the only file that uses it, though, so it would be quick to switch to a component instead of just a raw svg file.

Also, it will give us a chance to switch to exporting an inline `<svg>` rather than an `<img>`, as [accessibility experts recommend](https://www.sitepoint.com/tips-accessible-svg/), and fix the absent `alt=` problem on the `img` in this file by adding a customizable `title` attribute to the svg.